build/packages-template/bbb-html5/bbb-html5-conn-limit.conf

```diff
@@ -1 +0,0 @@
-limit_conn_zone $uri zone=ws_zone:5m;
```

Comment:
Is there a similar functionality now? 

> limit of 3 simultaneous connections (by uri) only for location /html5client/sockjs
https://github.com/bigbluebutton/bigbluebutton/pull/16771

Reply:
I think the graphql middleware theres a limitation per user, @gustavotrott can confirm it

Reply:
There is no limit, but I will take note to add it for Graphql ws connections!